homeassistant/components/ping/binary_sensor.py

```diff
@@ -213,7 +213,8 @@ async def async_ping(self):
                     out_error,
                 )
 
-            if pinger.returncode != 0:
+            if pinger.returncode > 1:
```

Comment:
Shouldn't this be >= 1? Since it would still log an exception now if the host is unreachable

Reply:
We don't want to log it if the code is 0 or 1. What am I missing ?

Reply:
I think I should have a coffee, you are right I was missing something. Everything checks out :+1: